packages/terra-search-field/src/SearchField.jsx

```diff
@@ -284,34 +297,44 @@ class SearchField extends React.Component {
       : undefined;
 
     return (
-      <div {...customProps} className={searchFieldClassNames}>
-        <div className={cx('input-group')}>
-          <input
-            {...additionalInputAttributes}
-            className={inputClass}
-            type="search"
-            placeholder={placeholder}
-            onChange={this.handleTextChange}
-            disabled={isDisabled}
-            aria-label={inputText}
-            aria-disabled={isDisabled}
-            onKeyDown={this.handleKeyDown}
-            onInput={this.handleInput}
-            ref={this.setInputRef}
-          />
-          {clearButton}
+      <div className={cx('search-container')}>
+        {isLabelVisible
+        && (
+        <label className={cx('label')}>{groupName}</label>
```

Comment:
Would it be better to wrap the input element inside the label?

Reply:
I think it is better to have the label separate from the input field. The idea is to group together the search components which would be the input field, the clear button and the search button. I feel the way it is now is also more readable.

Reply:
There are ways to associate the input to the label.  You can either wrap or there is an aria attribute that can be used.  However, we don't appear to be leveraging either.